<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TheGhost-PC" name="Test" tests="8" failures="3" timestamp="10 May 2015 17:32:51 GMT" time="0.458" errors="0">
  <testcase name="AcceptFriendRequest" time="0.056" classname="com.FCI.SWE.Controller.UserControllerTest"/>
  <testcase name="CreatePage" time="0.016" classname="com.FCI.SWE.Controller.UserControllerTest">
    <failure type="junit.framework.ComparisonFailure" message="expected:&amp;lt;Page [Has Been Created Successfully]&amp;gt; but was:&amp;lt;Page [With Same Name Has Been Created Before]&amp;gt;">
      <![CDATA[junit.framework.ComparisonFailure: expected:<Page [Has Been Created Successfully]> but was:<Page [With Same Name Has Been Created Before]>
	at junit.framework.Assert.assertEquals(Assert.java:100)
	at junit.framework.Assert.assertEquals(Assert.java:107)
	at com.FCI.SWE.Controller.UserControllerTest.CreatePage(UserControllerTest.java:28)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- CreatePage -->
  <testcase name="LikePage" time="0.012" classname="com.FCI.SWE.Controller.UserControllerTest">
    <failure type="junit.framework.ComparisonFailure" message="expected:&amp;lt;...n Liked Successfully[].&amp;gt; but was:&amp;lt;...n Liked Successfully[ ].&amp;gt;">
      <![CDATA[junit.framework.ComparisonFailure: expected:<...n Liked Successfully[].> but was:<...n Liked Successfully[ ].>
	at junit.framework.Assert.assertEquals(Assert.java:100)
	at junit.framework.Assert.assertEquals(Assert.java:107)
	at com.FCI.SWE.Controller.UserControllerTest.LikePage(UserControllerTest.java:35)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- LikePage -->
  <testcase name="MyHashStatstics" time="0.017" classname="com.FCI.SWE.Controller.UserControllerTest"/>
  <testcase name="SendFriendRequest" time="0.027" classname="com.FCI.SWE.Controller.UserControllerTest"/>
  <testcase name="SendMsg" time="0.061" classname="com.FCI.SWE.Controller.UserControllerTest"/>
  <testcase name="login" time="0.225" classname="com.FCI.SWE.Controller.UserControllerTest">
    <failure type="junit.framework.AssertionFailedError" message="expected:&amp;lt;OutboundJaxrsResponse{status=200, reason=OK, hasEntity=true, closed=false, buffered=false}&amp;gt; but was:&amp;lt;OutboundJaxrsResponse{status=200, reason=OK, hasEntity=true, closed=false, buffered=false}&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: expected:<OutboundJaxrsResponse{status=200, reason=OK, hasEntity=true, closed=false, buffered=false}> but was:<OutboundJaxrsResponse{status=200, reason=OK, hasEntity=true, closed=false, buffered=false}>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:86)
	at com.FCI.SWE.Controller.UserControllerTest.login(UserControllerTest.java:63)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="response" time="0.015" classname="com.FCI.SWE.Controller.UserControllerTest"/>
</testsuite> <!-- Test -->
